src/components/pages/devices/flyouts/deviceNew/deviceNew.js

```diff
@@ -129,16 +129,24 @@ const ProvisionedDevice = ({ device, t }) => {
 export class DeviceNew extends LinkedComponent {
   constructor(props) {
     super(props);
+
+    const { t } = this.props;
```

Comment:
Please remove this line, if `t` is declared on line 297.

Reply:
That's a completely different function. So its needed in both places unless I fully qualify it everywhere.